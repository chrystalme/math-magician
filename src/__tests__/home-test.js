import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Routes from '../components/routes';
import Home from '../components/home';

describe('Home', () => {
  test('Text match Home page test', () => {
    const { getByText } = render(<Routes><Home /></Routes>);
    const text = getByText('Math-magician');
    expect(text).toBeInTheDocument();
  });
  test('Test Home page snapshot', () => {
    const { getByText } = render(<Routes><Home /></Routes>);
    const text = getByText('Math-magician');
    expect(text).toMatchSnapshot();
  });
});
