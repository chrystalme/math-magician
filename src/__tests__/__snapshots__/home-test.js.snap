// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Test Home page snapshot 1`] = `
<a
  class="navbar-brand"
  href="/"
>
  Math-magician
</a>
`;
